speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
Yup, just tested it:

"Alexa, ask Kodi to play the movie Deepwater Horizon"

Response:

```Trying to match: the movie deep water horizon```

So, just to document it, it does appear that the lack of the article in the sample utterance when the requested movie isn't in the Slot is the problem.